library/src/CMakeLists.txt

```diff
@@ -22,26 +22,26 @@
 
 
 # ########################################################################
-# A helper function to prefix a source list of files with a common path into a new list (non-destructive)
+# A helper function to prefix a source list of files with a common path into a new list(non-destructive)
```

Comment:
Looks like this comment got caught in the automatic replacement

Reply:
Good catch.